### DISCUSSION TITLE
how to disable taskbar apps on startup

### DISCUSSION THREAD

**Discussion by jamescc10** [linuxmint/discussions#645] [Questions]
im using kde plasma with mint and i always see like 2 bluetooth icons, a update icon, etc in the taskbar
![image](https://github.com/user-attachments/assets/302adb30-3994-49f0-83d9-f6635f491a43)

**Comment by marco4mp** [comment]
What was the reason for installing KDE Plasma on Mint?

**Reply by jamescc10** [reply]
i dont like how cinnamon looks
